protected/views/patient/view.php

```diff
@@ -56,7 +56,7 @@
 			<div class="row">
 				<div class="large-12 column">
 					<div id="no-practice-address" class="alert-box alert with-icon">
-						Patient has no Practitioner practice address, please correct in PAS before printing Practitioner letter.
+						Patient has no Practitioner practice address, please correct by clicking the edit link in the patient summary before printing Practitioner letter.
```

Comment:
You might want to take a screenshot and send it to Jaclyn. I think she wanted to have a link in the banner to the edit page.